src/govuk/components/exit-this-page/exit-this-page.mjs

```diff
@@ -225,6 +232,8 @@ ExitThisPage.prototype.resetEscTimer = function () {
  */
 ExitThisPage.prototype.resetPage = function () {
   // If an overlay is set, remove it and reset the value
+  document.body.className = document.body.className.replace('govuk-exit-this-page-hide-content', '')
```

Comment:
We're already polyfilling `classList` at the top of the file. Is there a reason not to use it here? 

```suggestion
  document.body.classList.remove('govuk-exit-this-page-hide-content')
```